crates/move-stackless-bytecode/src/function_target_pipeline.rs

```diff
@@ -381,47 +376,33 @@ impl FunctionTargetsHolder {
             .or_default()
             .insert(FunctionVariant::Baseline, data);
 
-        if let Some(spec_attr) = func_env
+        if let Some(KnownAttribute::Verification(VerificationAttribute::Spec { focus, prove, target, no_opaque, ignore_abort })) = func_env
```

Comment:
does the prover report an error if there is a type, like `targets` instead of `target`?